src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="wrapper">
+        <legend>Personal information</legend>
+
+        <label class="fields">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            maxlength="8"
+          >
+        </label>
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="userName"
            autocomplete="off"
          >
```
yeah, opening and closing brackets should be at the same level, you're right
but if element has 2 attributes or more - start each one, including the first, on the new line with 2-space indentation related to tag